distribution/kernel/carbon-home/bin/wso2server.sh

```diff
@@ -267,6 +267,15 @@ fi
 START_EXIT_STATUS=121
 status=$START_EXIT_STATUS
 
+if [ -z "$JVM_MEM_OPTS" ]; then
+   java_version=$("$JAVACMD" -version 2>&1 | awk -F '"' '/version/ {print $2}')
+   JVM_MEM_OPTS="-Xms256m -Xmx1024m"
+   if [ "$java_version" \< "1.8" ]; then
```

Comment:
Do we need to check the jvm is less than  1.8, since we do not support for jvm's below 1.8?..